ihatemoney/templates/list_bills_archived.html

```diff
@@ -0,0 +1,180 @@
+{% extends "sidebar_table_layout.html" %}
```

Comment:
It seems to be a copy/paste from the more generic list_bills if I'm not mistaken. What about refactoring these two so it's easier to make edits to the way bills are rendered in general?